frontend/packages/data-portal/app/components/MenuDropdown.tsx

```diff
@@ -7,45 +7,60 @@ export function MenuDropdown({
   children,
   className,
   title,
+  variant = "standard",
+  buttonElement
 }: {
   children: ReactNode
+  variant?: 'standard' | 'outlined' | 'filled'
   className?: string
   title: ReactNode
+  buttonElement?: ReactNode
 }) {
   const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null)
   const menuRef = useRef<HTMLDivElement>(null)
 
+  const variantStyles = {
```

Comment:
I like this idea, it is similar to what I meant with the color checks for light/dark in the breadcrumbs